designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/theme/Color.kt

```diff
@@ -57,4 +57,5 @@ object Color {
     internal val CustomYellow: Color = Color(0xFFFADB14)
     internal val CustomBrown: Color = Color(0xFF7A130F)
     internal val CustomGray: Color = Color(0xFFDADADA)
+    internal val CustomGraySwitch: Color = Color(0xFFDADADA)
```

Comment:
isn't the same that the one above?

Reply:
You are completely correct, in my mind that grey was a darker one and I didn't even check. Correcting it right now